zeno_build/reporting/visualize.py

```diff
@@ -61,4 +76,28 @@ def pred(df, ops):
     )
     config = config.copy(update=zeno_config)
 
-    zeno(config)
+    zeno_instance = ZenoBackend(config)
+
+    # wait for the backend to complete processing
+    zeno_thread = zeno_instance.start_processing()
+    zeno_thread.join()
```

Comment:
Can we change the function definition of the start_processing function of ZenoBackend to return the thread variable? We will need to add `return self.__threads` after the [line](https://github.com/zeno-ml/zeno/blob/1eaa4ed30147f3781b810b9f2ebed27ee2d20487/zeno/backend.py#L231). This can help wait on the async thread working on `__process()` function.